### ISSUE TITLE
Ripple Color change Programmatically?

### ISSUE THREAD

**Issue by soban-shareviasms** [AnderWeb/discreteSeekBar#61]
How to change ripple color Programmatically?

**Comment by ignaciogs** [comment]
You can use the setThumbColor() and setScrubberColor() methods
```java
myDiscreteSeekBar.setScrubberColor(Color.GREEN);
myDiscreteSeekBar.setThumbColor(Color.GREEN, Color.GREEN);
```

**Comment by ignaciogs** [comment]
Implemented in commit: d8e2aa05df6636f2aaebd72864ad7e1d140d5aca
